pkg/vadmin/sandbox_sc_vc.go

```diff
@@ -63,6 +63,7 @@ func (v *VClusterOps) genSandboxSubclusterOptions(s *sandboxsc.Params, certs *HT
 	opts.SandboxName = s.Sandbox
 	opts.SCName = s.Subcluster
 	opts.SandboxPrimaryUpHost = s.UpHostInSandbox
+	opts.Imeta = true
```

Comment:
@roypaulin @cchen-vertica I got blocker now, we can't shutdown the special sandbox ( without meta-isolation and communal (prefix) isolation. Those can be create with the: "sls=false;imeta=false" options). I failed at `r.waitForSandboxUpgrade,`  step in replicateupgrade_reconciler.
```
 Normal   SandboxSubclusterStart        4m8s                   verticadb-operator  Starting add subcluster "pri1-sb" to sandbox "replica-group-b"
  Normal   SandboxSubclusterSucceeded    3m58s                  verticadb-operator  Successfully added subcluster "pri1-sb" to sandbox "replica-group-b"
  Normal   UpgradeStart                  3m55s (x2 over 5m16s)  verticadb-operator  Vertica server upgrade has started.
  Normal   IncompatibleUpgradeRequested  3m55s                  verticadb-operator  Requested upgrade is incompatible with the Vertica deployment. Falling back to offline upgrade.
  Warning  ClusterShutdownFailed         2m26s (x7 over 2m40s)  verticadb-operator  Failed to shutdown the sandbox replica-group-b
```
If I set `opts.Imeta = true` (the sandbox is not special now), we can shutdown the sandbox , but we can't promote sandbox to main since promotion is supported only for special sandboxes